Logger.Tests/FullNameTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Xunit;
+
+namespace Logger.Tests;
+
+public class FullNameTests
+{
+    [Theory]
+    [InlineData("First", "Last", "Middle", "First Middle Last")]
+    [InlineData("First", "Last", null, "First Last")]
+    public void FullName_ToString_HandlesNullMiddle(string first,  string last, string? middle, string expects)
+    {
+        FullName name = new(first, last,middle);
+        Assert.Equal(expects, name.ToString());
```

Comment:
```suggestion
    public void FullName_ToString_HandlesNullMiddle(string first,  string last, string? middle, string expected)
    {
        FullName name = new(first, last,middle);
        Assert.Equal(expected, name.ToString());
```
I think slightly better naming